pgx_unix/src/pgx_unix.ml

```diff
@@ -19,11 +19,12 @@
  * Boston, MA 02111-1307, USA.
 *)
 
+external reraise : exn -> _ = "%reraise"
+
 module Simple_thread = struct
   type 'a t = 'a
   let return x = x
   let (>>=) v f =  f v
-  let fail = raise
```

Comment:
Why did we do this alias before?

Reply:
No idea.. this was before my time. I just noticed that it's completely useless now :\